auth_otp/views.py

```
@@ -0,0 +1,357 @@
+            
+            return Response(
+                {
+                    'success': False,
+                    'error': result.get('error'),
+                    'message': result.get('message'),
+                    'details': result
+                },
+                status=status_code
+            )
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)